commands/bug.go

```diff
@@ -17,8 +19,16 @@ func Version() string {
 	return resp
 }
 
-func GetBug(id string) Defects {
-	url := fmt.Sprintf("bug/%v", id)
+// GetBug : Fetch a bug/s matching an id or a set of keywords. Also supports adding a limit of results.
+func GetBug(filter, limit string) Defects {
+	var url string
+
+	if matched, _ := regexp.MatchString("^[0-9]+$", filter); matched {
```

Comment:
Can you add a comment about what the regex is looking for here and the else statement below? I understand that it is looking for numbers (digits) but it will be useful to document it for the future.